README.md

```diff
@@ -325,34 +342,27 @@ createdb FoodDatabase
 
 2. Open a new terminal window and go to your `Package.swift` file.
 ```
-cd FoodTrackerBackend/FoodServer
+cd ~/FoodTrackerBackend/FoodServer
 open Package.swift
 ```
 3. Add the Swift-Kuery-ORM and Swift-Kuery-PostgreSQL packages.
 ```swift
 .package(url: "https://github.com/IBM-Swift/Swift-Kuery-ORM.git", .upToNextMinor(from: "0.0.1")),
-.package(url: "https://github.com/IBM-Swift/Swift-Kuery-PostgreSQL.git", .upToNextMinor(from: "1.1.0")),
+.package(url: "https://github.com/IBM-Swift/Swift-Kuery-PostgreSQL.git", from: "1.1.0"),
 ```
-below the line `.package(url:
-  "https://github.com/IBM-Swift/Health.git", from: "0.0.0"),`
+below the line `.package(url: "https://github.com/IBM-Swift/Health.git", from: "0.0.0"),`
```

Comment:
As we are finding out right now, this should be specifying at least version `1.0.0`